stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -278,6 +278,11 @@ spec:
               containerPort: {{ .Values.metrics.port  }}
           resources:
 {{ toYaml .Values.metrics.resources | indent 12 }}
+          {{- if .Values.metrics.securityContext.enabled }}
+          securityContext:
+            runAsUser: {{ .Values.metrics.securityContext.runAsUser }}
+            fsGroup: {{ .Values.metrics.securityContext.fsGroup }}
+          {{- end }}
           livenessProbe:
```

Comment:
The liveness Probe also needs double dash or else it fails with error "unrecognized shorthand flag -m" when authentication is enabled.

Reply:
Good catch! I will create a bugfix asap!